src/client/driver/command-executors/execute-action-command.js

```diff
@@ -201,7 +202,12 @@ export default function executeActionCommand (command, elementAvailabilityTimeou
 
                 xhrBarrier = new XhrBarrier();
 
-                return createAutomation(elements, command).run();
+                var automation = createAutomation(elements, command);
+
+                if (automation)
+                    return createAutomation(elements, command).run();
+
+                return delay(command.type === COMMAND_TYPE.wait ? command.timeout : 0);
```

Comment:
We should implement `wait` server side, so it will not be interrupted by page loads